src/components/ADempiere/FieldDefinition/FieldSearch/Invoice/PanelForm/QueryCriteria/businessPartner.vue

```diff
@@ -0,0 +1,147 @@
+<!--
+ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+Copyright (C) 2018-Present E.R.P. Consultores y Asociados, C.A. www.erpya.com
+Contributor(s): Edwin Betancourt EdwinBetanc0urt@outlook.com https://github.com/EdwinBetanc0urt
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <https:www.gnu.org/licenses/>.
+-->
+
+<template>
+  <el-form-item
+    :label="$t('field.invoice.businessPartner')"
+  >
+    <el-select
+      v-model="currentValue"
+      filterable
+      clearable
+      remote
+      :remote-method="remoteMethod"
+      @visible-change="showListBP"
+    >
+      <empty-option-select
+        :current-value="currentSaleOrder"
+        :is-allows-zero="false"
+      />
+      <el-option
+        v-for="(option, key) in optionsList"
+        :key="key"
+        :value="option.id"
+        :label="option.displayColumn"
```

Comment:
Implement as `:label="item.values.DisplayColumn"` to improve peformance

Reply:
https://github.com/solop-develop/frontend-core/pull/2220/commits/9531be9e2962182759a5db75cd377bcda4bb6ece